src/Instance/ItemsList/utils.js

```diff
@@ -1,21 +1,53 @@
-import {
-  orderBy,
-} from 'lodash';
+/**
+ * enumerationSorter
+ * Parse an object's enumeration value numerically
+ * Given { enumeration: [string] value } where value is a string like
+ * "v.23" or "v.23:no.11-12", compare "23" and "11".
+ * If the value does not match the pattern, compare the full string.
+ *
+ * @param  {[object]} an object containing the property "enumeration".
+ */
+const enumerationSorter = (a, b) => {
+  const aMatches = a.enumeration.match(/^v\.([0-9]+)(:no\.([0-9]+).*)?$/);
+  const bMatches = b.enumeration.match(/^v\.([0-9]+)(:no\.([0-9]+).*)?$/);
+  if (aMatches && bMatches) {
+    if (aMatches[1] === bMatches[1]) {
+      return Number.parseFloat(aMatches[3], 10) - Number.parseFloat(bMatches[3], 10);
+    } else {
+      return Number.parseFloat(aMatches[1], 10) - Number.parseFloat(bMatches[1], 10);
+    }
+  }
+
```

Comment:
Indeed, it makes sense if there is a strict common pattern for `item.enumeration`. If not - 🤷‍♂️  .
In any case, we'd better wait for your question to be answered in the [comments](https://issues.folio.org/browse/UIIN-1412?focusedCommentId=97506&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-97506) to this ticket.